benchmarks/web-serving/db_server/Dockerfile

```diff
@@ -1,30 +1,12 @@
-FROM ubuntu:14.04
+FROM cloudsuite/mysql:mariadb-10.3
 LABEL maintainer="Mark Sutherland <mark.sutherland@epfl.ch>"
 
 ENV root_password root
-ENV web_host web_server 
 
-# Install MySQL
-RUN echo mysql-server mysql-server/root_password password ${root_password} | sudo debconf-set-selections
-RUN echo mysql-server mysql-server/root_password_again password ${root_password} | sudo debconf-set-selections
-
-RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install -y \
-	mysql-server
-
-# Allow it to listen from outer world
-RUN sed -i -e"s/^bind-address\s*=\s*127.0.0.1/bind-address = 0.0.0.0/" /etc/mysql/my.cnf
```

Comment:
Please check if it is still necessary if we use mariadb.

Reply:
@yunho-oh - Similarly, here as well, this is already removed, as it was not needed.

Reply:
Sorry, never mind. My comment is wrong.